docs/rule/no-nested-splattributes.md

```diff
@@ -0,0 +1,30 @@
+# no-nested-splattributes
+
+Having `...attributes` on multiple elements nested within each other in a
+component can cause unintended results.
+
+This rule prevent you from running into this issue by disallowing
+`...attributes` if any of the parent elements already has `...attributes` on it.
+
+## Examples
+
+This rule **forbids** the following:
+
+```hbs
+<div ...attributes>
+  <div ...attributes>
+    ...
+  </div>
+</div>
+```
+
+This rule **allows** the following:
+
+```hbs
+<div ...attributes>...</div>
+<div ...attributes>...</div>
+```
+
+## Migration
+
+- Remove the inner `...attributes` declaration
```

Comment:
Thought about that too, but I think it's probably safer to let the user decide which splattribute he wants to remove. Most likely it will be the inner one, but probably not in 100% of the cases.

Reply:
Ya, makes sense.

Reply:
I agree that we should not make assumptions about which `...attributes` is the mistaken one (in fact, they could both be legit).